### PR TITLE
style: clj kondo fixes

### PR DESCRIPTION
**Pull request by jsmassa**
Doesn't introduce any changes in behavior, but  resolves distracting clj-kondo warnings

### CODE REVIEW

**Comment by jsmassa** [comment]
Should be merged after #90 since it includes the changes

**Review by whilo — APPROVED** [review]
Looks good. The old file store is still there because it tests the migration code, which we cannot remove in general, because somebody might still upgrade from an old konserve version to a new one (unless we want to break these setups or can somehow explicitly tell them to upgrade to some intermediate version first).

File: .clj-kondo/hooks/go_try_.clj [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  (:require [clj-kondo.hooks-api :as api]))
+
+(defn go-try- [{:keys [:node]}]
+  (let [new-node (api/list-node
+                  (apply list
+                         (concat
+                          [(api/token-node 'try)]
+                          (rest (:children node))
+                          [(api/list-node (list (api/token-node 'finally)))])))]
+    {:node new-node}))
```
Comment (TimoKramer) [inline .clj-kondo/hooks/go_try_.clj:11]: Don't fully understand it but cool 👍

**Review by TimoKramer — APPROVED** [review]

File: src/konserve/compliance_test.cljc [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
-            [konserve.memory :refer [new-mem-store]]))
+            #?(:cljs [cljs.test :refer [is]])
+            #?(:clj [clojure.test :refer [are is testing]])))
 
-(deftype UnknownType [])
+#_(deftype UnknownType [])
 
-#?(:clj (defn exception? [thing]
+#_(:clj (defn exception? [thing]
           (instance? Throwable thing)))
```
[the remark this answers is not in the document]
Comment (jsmassa) [reply src/konserve/compliance_test.cljc:10]: The two functions are being used in the test below which is commented out.

**Review by TimoKramer — APPROVED** [review]

**Pull request merged**